migration/v4/model/applicationtag.go

```diff
@@ -0,0 +1,19 @@
+package model
+
+//
+// ApplicationTag represents a row in the join table for the
+// many-to-many relationship between Applications and Tags.
+type ApplicationTag struct {
```

Comment:
hint: necessary to bring this forward since it has a relationship to a model that was updated in this migration (Application)